src/rust/demikernel/libos/memory.rs

```diff
@@ -54,7 +52,7 @@ impl MemoryLibOS {
     pub fn create_pipe_for_push(&mut self, name: &str) -> Result<QDesc, Fail> {
```

Comment:
Can you delete this function and update the pipe examples in our test suite?

Reply:
Done.